cmd/calyptia/create_pipeline_test.go

```diff
@@ -40,7 +40,7 @@ func Test_newCmdCreatePipeline(t *testing.T) {
 	cmd.SilenceErrors = true
 	cmd.SilenceUsage = true
 	cmd.SetArgs([]string{
-		"--core_instance", "want_core_instance",
+		"--core-instance", "want_core-instance",
```

Comment:
As these are parameters we probably should update the `want_` prefixes for the IDs. It's confusing.

It happens a few times in the codebase so I've only raised one comment.